components/lcd/esp_lcd_gc9503/idf_component.yml

```diff
@@ -1,8 +1,8 @@
-version: "2.1.0"
+version: "3.0.0"
 targets:
   - esp32s3
 description: ESP LCD GC9503
 url: https://github.com/espressif/esp-bsp/tree/master/components/lcd/esp_lcd_gc9503
 dependencies:
-  idf: ">=5.0.4"
```

Comment:
@espzav I also make a change here. Because it relies on a new feature of IDF release/v5.0/v5.1 and master. PTAL!